aep/general/0135/aep.md.j2

```diff
@@ -4,15 +4,45 @@ In REST APIs, it is customary to make a `DELETE` request to a resource's URI
 (for example, `/v1/publishers/{publisher}/books/{book}`) in order to delete
 that resource.
 
-Resource-oriented design (AIP-121) honors this pattern through the `Delete`
-method. These RPCs accept the URI representing that resource and usually return
+Resource-oriented design ([AEP-121][aep-121]) honors this pattern through the `Delete`
+method. These operations accept the URI representing that resource and usually return
```

Comment:
Should "operations" be "methods"?  I think the AIPs more or less use them interchangeably; should the AEPs be more careful about this?

Reply:
I'm fine to use "method" over "operation" in most cases, but I think there will be cases ("long running operations" comes to mind) where "operation" seems like the better term. I'll change attempt to change "operation" to method where it seems right to me and then we can assess what's left.

Reply:
I think we decided to go with "operations" over "methods" but that will need to be a more global change, so I'll leave that for a future PR.